src/components/widgets/WebWidget.tsx

```diff
@@ -0,0 +1,39 @@
+import { useNavigation } from '@react-navigation/core';
+import React, { useCallback } from 'react';
+
+import { Icon, normalize, texts } from '../../config';
+import { ScreenName, WidgetProps } from '../../types';
+import { Image } from '../Image';
+
+import { DefaultWidget } from './DefaultWidget';
+
+export const WebWidget = ({ text, additionalProps }: WidgetProps) => {
+  const navigation = useNavigation();
+
+  const onPress = useCallback(() => {
+    navigation.navigate(ScreenName.Web, {
+      title: additionalProps?.staticContentTitle ?? texts.widgets.web,
```

Comment:
how does keeping `texts.widgets.web` here is helpful? it is an empty string

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/932/commits/23d95531fe800d088941b71c9db17cc644b60732